### ISSUE TITLE
Set dojoConfig via a Rails helper

### ISSUE THREAD

**Issue by robin850** [robin850/dojo-rails#2]
Hi everyone,

I open this ticket to ask you if the idea of creating a Rails helper to set `dojoConfig` sounds good. It will behaves just like this :

``` erb
<%= dojo_config :base_url => "/foo/bar" %>
```

And will be converted to Javascript like that :

``` javascript
dojoConfig = { baseUrl: "/foo/bar" }
```

By the way, the code of the helper will be very concise :

``` ruby
require 'json'

module DojoHelper
  def dojo_config(options = {})
    options.keys.each do |k|
      new_key = k.camelize(:lower)
      options[new_key] = options[k]
      options.delete(k)
    end

    options.to_json
  end
end
```

What do you guys think ?

Have a nice day.

/cc @NikoRoberts

**Comment by NikoRoberts** [comment]
sounds good to me

**Comment by robin850** [comment]
Ok, thanks for the feedback ! :)

**Comment by jamespedid** [comment]
This sounds like a nice idea.

**Comment by robin850** [comment]
Thanks @jamespedid, I'm working on it!

**Comment by robin850** [comment]
Hi everyone, I finally released this feature. This will be available in 0.14.0 (which will be the final 0.x release before 1.0.0). Thanks for your feedback!
